### DISCUSSION TITLE
Is it possible to make a panel menu item enabled by default?

### DISCUSSION THREAD

**Discussion by BoisAuLit** [primefaces/community#1557] [PrimeNG] [answered]
When using panel menu component, I cannot make a menu item selected (active) by default. I'd like a menu item to be selected without selecting it explicitly on mouse click when the menu shows on the page, such as below:
![Screenshot 2024-04-01 at 00 08 14](https://github.com/primefaces/community/assets/18257978/5466c7e9-d458-438a-b0fb-04f3d63d854e)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately it's not possible with the menu components.

**Reply by BoisAuLit** [reply]
@cetincakiroglu  Thanks for the clarification.
